### PR TITLE
Issue 271

### PR DESCRIPTION
**Pull request by aeschylus**

### CODE REVIEW

File: app/models/embed/purl.rb [hunk 1]
```diff
@@ -13,9 +13,9 @@ def title
 
     def type
       @type ||= begin
-        contentMetadata = ng_xml.xpath('//contentMetadata').first
-        contentMetadata.attributes['type'].try(:value) if contentMetadata.present?
-      end
+                  contentMetadata = ng_xml.xpath('//contentMetadata').first
```
Comment (mejackreed) [inline app/models/embed/purl.rb]: Do we need to modify this whitespace here?

File: app/models/embed/purl.rb [hunk 2]
```diff
@@ -42,10 +42,10 @@ def license
 
     def embargo_release_date
       @embargo_release_date ||= begin
-        if embargoed?
-          ng_xml.xpath('//rightsMetadata/access[@type="read"]/machine/embargoReleaseDate').try(:text)
-        end
-      end
+                                  if embargoed?
```
Comment (mejackreed) [inline app/models/embed/purl.rb]: Same about whitespace

File: app/models/embed/purl.rb [hunk 3]
```diff
@@ -96,20 +102,20 @@ def purl_xml_url
...
-        end
-        raise ResourceNotAvailable unless response.success?
-        response.body
-      rescue Faraday::Error::ConnectionFailed => error
-        nil
-      rescue Faraday::Error::TimeoutError => error
-        nil
-      end
+                      conn = Faraday.new(url: purl_xml_url)
+                      response = conn.get do |request|
```
Comment (mejackreed) [inline app/models/embed/purl.rb]: Are we moving this for a reason?

File: spec/models/embed/envelope_spec.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+describe Embed::Envelope do
+  include PURLFixtures
+  describe 'to_bounding_box' do
```
Comment (mejackreed) [inline spec/models/embed/envelope_spec.rb]: Could we dry up these spec using rspec's `let`? We can go over an example.

File: app/models/embed/purl.rb [hunk 5]
```diff
@@ -183,9 +189,14 @@ def has_image_data?
...
           @file.xpath('./location[@type="url"]').present?
         end
+        def has_geo_data?
+          @file.xpath('//*[name()="gml:Envelope"]').present?
+        end
         def preview_types
           ["image/jp2"]
         end
+        def bounding_points 
+        end
```
Comment (aalsum) [inline app/models/embed/purl.rb]: Remove bounding_points function

File: app/models/embed/purl.rb [hunk 6]
```diff
@@ -183,9 +189,14 @@ def has_image_data?
         def has_location_data?
           @file.xpath('./location[@type="url"]').present?
         end
+        def has_geo_data?
```
Comment (mejackreed) [inline app/models/embed/purl.rb]: I think this method is noop too. Either should be moved and tested/used, or removed

**Comment by mejackreed** [comment]
@aeschylus https://github.com/sul-dlss/sul-embed/pull/278 should fix the errors with the JS tests that are failing.

https://github.com/sul-dlss/sul-embed/pull/276 refactors/drys up some of the specs in this PR

**Comment by aeschylus** [comment]
I'm not getting these failing tests locally, and I believe the travis updates have already been applied.

**Pull request merged**